data/data-pipeline/data_pipeline/etl/sources/epa_rsei_aggregate/etl.py

```diff
@@ -0,0 +1,268 @@
+        logger.info("Starting 2.5 MB data download.")
+
+        unzip_file_from_url(
+            file_url=self.AGGREGATED_RSEI_SCORE_FILE_URL,
+            download_path=self.TMP_PATH,
+            unzipped_file_path=self.TMP_PATH / "epa_rsei_aggregated_microdata",
+        )
+
+        self.df = pd.read_csv(
+            filepath_or_buffer=self.TMP_PATH
+            / "epa_rsei_aggregated_microdata"
+            / "CensusMicroTracts2019_2019_aggregated.csv",
+            # The following need to remain as strings for all of their digits, not get
+            # converted to numbers.
+            low_memory=False,
+            header=0,
+        )
+
+    def transform(self) -> None:
+        logger.info("Starting transforms.")
+
+        # the column headers from the above dataset are actually a census tract's data at this point
+        # We will use this data structure later
+        input_columns = [
+            self.TRACT_INPUT_COLUMN_NAME,
+            self.NUMBER_FACILITIES_INPUT_FIELD,
+            self.NUMBER_RELEASES_INPUT_FIELD,
+            self.NUMBER_CHEMICALS_INPUT_FIELD,
+            self.AVERAGE_TOXICITY_INPUT_FIELD,
+            self.SCORE_INPUT_FIELD,
+            self.POPULATION_INPUT_FIELD,
+            self.CSCORE_INPUT_FIELD,
+            self.NCSCORE_INPUT_FIELD,
+        ]
+        # The subsequent step make the current columns the first row
+        # of the dataframe and adds it as an individual sample
+        first_row = dict(zip(input_columns, self.df.columns))
+        self.df.columns = input_columns
+        # add first row to dataframe
+        row_df = pd.DataFrame(first_row, index=[0]).reset_index(drop=True)
+        # simply concatenate both dataframes
+        self.df = pd.concat([row_df, self.df])
+
+        score_columns = [x for x in self.df.columns if "SCORE" in x]
+
+        # coerce dataframe type to perform correct next steps
+        self.df[score_columns] = self.df[score_columns].astype(float)
+
+        self.df.rename(
+            columns={
+                self.TRACT_INPUT_COLUMN_NAME: self.GEOID_TRACT_FIELD_NAME,
+                self.NUMBER_FACILITIES_INPUT_FIELD: field_names.EPA_RSEI_NUMBER_FACILITIES_OUTPUT_FIELD,
+                self.NUMBER_RELEASES_INPUT_FIELD: field_names.EPA_RSEI_NUMBER_RELEASES_OUTPUT_FIELD,
+                self.NUMBER_CHEMICALS_INPUT_FIELD: field_names.EPA_RSEI_NUMBER_CHEMICALS_OUTPUT_FIELD,
+                self.AVERAGE_TOXICITY_INPUT_FIELD: field_names.EPA_RSEI_AVERAGE_TOXICITY_OUTPUT_FIELD,
+                self.SCORE_INPUT_FIELD: field_names.EPA_RSEI_SCORE_OUTPUT_FIELD,
+                self.CSCORE_INPUT_FIELD: field_names.EPA_RSEI_CSCORE_OUTPUT_FIELD,
+                self.NCSCORE_INPUT_FIELD: field_names.EPA_RSEI_NCSCORE_OUTPUT_FIELD,
+                self.POPULATION_INPUT_FIELD: field_names.EPA_RSEI_POPULATION_OUTPUT_FIELD,
+            },
+            inplace=True,
+        )
+
+        # 5th, 25th, 50th, 75th and 99th percentiles for each score column
```

Comment:
How are you planning to use these bucketed "cut" columns? There's a lot of code used to create them, but we don't actually use them anywhere or produce any analysis based on them, so I'm wondering if making the code for this ETL class that much more complex is worth it if we never really use the results. Overall the class could be a lot cleaner if just output some of the scores.

Reply:
The rationale is outline [here]( https://github.com/usds/justice40-tool/pull/1101/files#diff-b3ec2dbf735f5d657d3212c93e05e5bd3d6de42a4b0db7e7bcd20aa9f51e5829R185) and also in introductory comments [here](https://github.com/usds/justice40-tool/pull/1101#issue-1094744768)  (specifically #3 and #4).

I wanted to discuss this with you as stated [here](https://github.com/usds/justice40-tool/pull/1101/files#r779188783).

Reply:
Ok. The ratio of complexity to add these fields to the value they bring seems too high to me – I don't think it's worth it based on what I'm seeing it. We do not use these results anywhere that I can tell. 

Could you remove the code that creates these buckets for now? It'll be in our git history so we can always resuscitate them if necessary.